Resources/Data Sets/index_nl.json

```
@@ -764,6 +764,20 @@
                 "kind": "file",
                 "associated_datafile": "Fear of Statistics.csv"
               }
```

```suggestion
              },
```
here needs to be comma for the json to work